RFC-010-Omni-Layer.md

```diff
@@ -0,0 +1,112 @@
+# The Omni Layer Asset
+
+- RFC-Number: 010
+- Status: Draft
+- Discussion-issue: -
+- Created on: 27 Mar. 2019
+
+**Table of Contents**
+
+  * [Description](#description)
+  * [The Omni Layer Assets](#the-omni-layer-assets)
+    * [`quantity`](#quantity)
+    * [`property_id`](#property_id)
+  * [Registry extension](#registry-extension)
+  * [Examples](#examples)
+
+## Description
+
+This RFC defines how Bitcoin-based Omni Layer Assets are described in Asset type headers.
+Omni Layer refers specifically to the [Omni Layer Protocol](https://github.com/OmniLayer/spec).
+
+The Asset type header was introduced in [RFC002](./RFC-002-SWAP.md) to describe assets being exchanged in a COMIT SWAP protocol.
+
+This in combination with subsequent RFCs that define concrete execution steps for particular SWAP protocols enables users to negotiate and execute Omni Layer asset swaps.
```

Comment:
🔧 I'd prefer not to have this sentence. This RFC can just describe Omin layer assets and not promise what other RFCs are going to do.